ddtelemetry/src/worker/mod.rs

```diff
@@ -183,6 +183,9 @@ mod serialize {
 }
 
 impl TelemetryWorker {
+    // Default request timeout in milliseconds.
+    const DEFAULT_TIMEOUT: u64 = 10_000;
```

Comment:
This could be imported from the Endpoint timeout, avoid duplicating the same constant